### PR TITLE
fixing 2 missing route decorations

### PR DESCRIPTION
**Pull request by archtaurus**

### CODE REVIEW

**Comment by archtaurus** [comment]
Makes EarlyRefresh and ExpireRefresh diffrent.

图片

File: flask_praetorian/exceptions.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ class EarlyRefreshError(PraetorianError):
     """
     The jwt token has not yet expired for access and may not be refreshed
     """
-    pass
+    status_code = 425  # HTTP Status Code : 425 Too Early
```
Comment (dusktreader) [inline flask_praetorian/exceptions.py:37]: Oh, nice work!

**Review by dusktreader — APPROVED** [review]
Looks great! Thanks for the submission

**Comment by dusktreader** [comment]
I need to examine why travis ci isn't running my checks right now. it should be mergable once that's working

**Pull request merged**